packages/components/atoms/f-form-field/README.md

```diff
@@ -15,73 +15,108 @@ Fozzie Form Field Component.
 [![Coverage Status](https://coveralls.io/repos/github/justeat/f-form-field/badge.svg)](https://coveralls.io/github/justeat/f-form-field)
 [![Known Vulnerabilities](https://snyk.io/test/github/justeat/f-form-field/badge.svg?targetFile=package.json)](https://snyk.io/test/github/justeat/f-form-field?targetFile=package.json)
 
+---
 
 ## Usage
 
-1.  Install the module using NPM or Yarn:
+### Installation
+
+Install the module using NPM or Yarn:
 
-    ```bash
-    yarn add @justeat/f-form-field
-    ```
+```sh
+yarn add @justeat/f-form-field
+```
 
-    ```bash
-    npm install @justeat/f-form-field
-    ```
+```sh
+npm install @justeat/f-form-field
+```
 
-2.  Import the component
+### Vue Applications
 
-    You can import it in your Vue SFC like this (please note that styles have to be imported separately):
+You can import it in your Vue SFC like this (please note that styles have to be imported separately):
 
-    ```
-    import FormField from '@justeat/f-form-field';
-    import '@justeat/f-form-field/dist/f-form-field.css';
+```js
+import FormField from '@justeat/f-form-field';
+import '@justeat/f-form-field/dist/f-form-field.css';
 
-    export default {
-        components: {
-            FormField
-        }
+export default {
+    components: {
+        FormField
     }
-    ```
+}
+```
 
-    If you are using Webpack, you can import the component dynamically to separate the header bundle from the main `bundle.client.js`:
+If you are using Webpack, you can import the component dynamically to separate the `mega-modal` bundle from the main `bundle.client.js`:
 
-    ```
-    import '@justeat/f-form-field/dist/f-form-field.css';
+```js
+import '@justeat/f-form-field/dist/f-form-field.css';
 
-    export default {
-        components: {
-            ...
-            FormField: () => import(/* webpackChunkName: "form-field" */ '@justeat/f-form-field')
-        }
+export default {
+    components: {
+        // …
+        FormField: () => import(/* webpackChunkName: "form-field" */ '@justeat/f-form-field')
     }
+}
+```
+
+## Configuration
+
+### Props
+
+There may be props that allow you to customise its functionality.
+
+The props that can be defined are as follows (if any):
+
+| Prop  | Type  | Default | Description |
+| ----- | ----- | ------- | ----------- |
+
+### CSS Classes
+
+The modal has its own styles which are scoped to the component using CSS modules to prevent conflicts with existing styles on the page.
+
+In addition to this, the modal exposes some classes which you can target in your application.
```

Comment:
```suggestion
The form-field has its own styles which are scoped to the component using CSS modules to prevent conflicts with existing styles on the page.

In addition to this, the form-field exposes some classes which you can target in your application.
```